### DISCUSSION TITLE
Telephony working?

### DISCUSSION THREAD

**Discussion by Laanan** [GloDroid/glodroid_manifest#163] [Q&A]
Hi folks,

Wondering if telephony/4g is working yet for Glodroid. Also wondering how I may contribute. Really wanting to see the day that my Pinephone is functional. ;-) Thanks!

**Comment by rsglobal** [comment]
Starting from not yet released v0.7.0 and android-12 there is some progress with calls.
Unfortunately due to bugs with graphics it's not yet released.
From the latest news you can do/accept calls and audio is working.
But there are a lot of issues left.
